runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java

```diff
@@ -44,7 +44,8 @@
     // We are temporarily naming this stream with the same name as TRANSACTION_STREAM_ID to avoid breaking LR code
     // while transition to UFO is completed (once migration is complete to UFO this stream can be renamed)
     // add a prefix to the name: e.g., org.corfudb.logreplication.transactionstream
-    public static final UUID LOG_REPLICATOR_STREAM_ID = CorfuRuntime.getStreamID("Transaction_Stream");
+    public static final String LOG_REPLICATOR_STREAM_NAME = "Transaction_Stream";
```

Comment:
Note, since TRANSACTION_STREAM_ID is deprecated, I left it as is.

Reply:
Actually this is not the old transaction stream. It is a small trick for log replication compatibility.